pages/index.js

```diff
@@ -25,12 +49,21 @@ const Main = styled.main`
 
 const Heading = styled.h1`
   text-align: center;
-  margin: 2rem;
+  margin: 61px 30px 50px;
 `;
 
-const AnswerListItems = styled.li`
-  list-style: none;
-  margin: 20px;
-  margin-left: -20px;
-  max-width: 250px;
+const CurrentDate = styled.p`
+  margin-bottom: 120px;
+`;
+
+const StyledLink = styled.a`
```

Comment:
See my comment above regarding the `Link` component.
```suggestion
const StyledLink = styled(Link)`
```